build.sbt

```
@@ -116,3 +118,11 @@ libraryDependencies ++= Seq(
   compilerPlugin("com.github.ghik" %% "silencer-plugin" % "1.3.1"),
   "com.github.ghik" %% "silencer-lib" % "1.3.1" % Provided
 )
+
+// For sbt plugin sbt-buildinfo
+lazy val root = (project in file(".")).
```

```suggestion
enablePlugins(BuildInfoPlugin)
buildInfoKeys := Seq[BuildInfoKey](name, version, scalaVersion, sbtVersion)
buildInfoPackage := "co.ledger.wallet.daemon"
```

We are using flat style sbt file (no project definition), so do like this

---

This is the recommended way of using this plugin, see https://github.com/sbt/sbt-buildinfo#usage

If I remove this level, the compilation step fails because of BuildInfo

---

Tested on my machine, the compilation won't fail.